File: iniciante/python/1180-menor-e-posicao.py
def menor_e_posicao():
    lista_de_valores = entrada()
    valor_minimo = min(lista_de_valores)
    posicao = lista_de_valores.index(valor_minimo)
    print(f'Menor valor: {valor_minimo}\n'
          f'Posicao: {posicao}')


def entrada():
    n = int(input())
    valores = list(map(int, input().split()))
    return valores


menor_e_posicao()

File: iniciante/c/1180-menor-e-posicao.c
#include <stdio.h>

int main(){
    int tamanho, i, menor, posicao = 0;
    scanf("%d", &tamanho);
    
    int v[tamanho];
    for(i = 0; i < tamanho; i++){
        scanf("%d", &v[i]);
    }

    menor = v[0];
    for(i = 1; i < tamanho; i++){
        if (v[i] < menor){
            menor = v[i];
            posicao = i;
        }
    }
    printf("Menor valor: %d\n", menor);
    printf("Posicao: %d\n", posicao);

    return 0;
}
